source/modules/MEI.shared.xml

```diff
@@ -7495,14 +7534,18 @@
     </content>
     <constraintSpec ident="check_respStmt" scheme="schematron">
       <constraint>
-        <sch:rule context="mei:respStmt[not(ancestor::mei:change)]">
+        <sch:rule context="mei:respStmt[not(ancestor::mei:change) and not(ancestor::mei:work)]">
           <sch:assert
             test="(mei:resp and (mei:name or mei:corpName or mei:persName)) or
             count(mei:*[@role]) = count(mei:*) and count(mei:*) &gt; 0"
             role="warning">At least one element pair (a resp element and a name-like element) is
             recommended. Alternatively, each name-like element may have a @role
             attribute.</sch:assert>
         </sch:rule>
+        <sch:rule context="mei:respStmt[ancestor::mei:work]">
+          <sch:report test="count(mei:*[@role]) &lt; count(mei:*) or count(mei:*) = 0" role="warning">Name-like elements with a @role are recommended here.</sch:report>
+          <sch:report test="mei:resp" role="warning">Name-like elements with a @role are recommented here (instead of resp).</sch:report>
```

Comment:
```suggestion
          <sch:report test="mei:resp" role="warning">Name-like elements with a @role are recommended here (instead of resp).</sch:report>
```